### PR TITLE
feat: add unit tests for lambdas

### PR DESCRIPTION
**Pull request by maxneuvians**
This PR adds some basic unit testing for the lambdas we use to collect metrics. Ideally these would be somewhere else and we would just push in a bundled artifact into the terraform. We will iterate towards that :). The unit tests are not comprehensive in that it does not cover all the edge cases. This is however a step in getting there. Noticed one bug and a few things we may want to talk over.

### CODE REVIEW

File: .github/workflows/lambda-tests.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: Test lambdas
+        working-directory: ./server/aws/lambda
+        run: |
+          npm install
```
Comment (CalvinRodo) [inline .github/workflows/lambda-tests.yml:16]: Should be `npm ci` so we don't accidentally upgrade package versions in CI 

https://docs.npmjs.com/cli/v7/commands/npm-ci

**Review by CalvinRodo — APPROVED** [review]
One small thing, but nothing that is a blocker at all and can be addressed in a future PR.

File: .github/workflows/lambda-tests.yml (hunk 1 above)
Comment (maxneuvians) [reply .github/workflows/lambda-tests.yml:16]: Nice catch

**Comment by CalvinRodo** [comment]
Closes #159
